packages/widget/tests/unit/components/transfer/fungible/fungible-token-transfer.test.ts

```diff
@@ -101,4 +112,71 @@ describe('Fungible token Transfer', function () {
 
     assert(sygmaAddressInput.address === '');
   });
+
+  it('should filter whitelisted networks and resources', async () => {
+    const whitelistedSourceNetworks = ['cronos'];
+    const whitelistedDestinationNetworks = ['sepolia'];
+    const whitelistedResources = ['ERC20LRTest'];
+    const connectedAddress = '0x95222290DD7278Aa3Ddd389Cc1E1d165CC4BAfe5';
+    const walletContext = await fixture<WalletContextProvider>(html`
+      <sygma-wallet-context-provider></sygma-wallet-context-provider>
+    `);
+
+    const fungibleTransfer = await fixture<FungibleTokenTransfer>(
+      html`<sygma-fungible-transfer
+        .whitelistedSourceNetworks=${whitelistedSourceNetworks}
+        .whitelistedDestinationNetworks=${whitelistedDestinationNetworks}
+        .whitelistedSourceResources=${whitelistedResources}
+        .environment=${Environment.TESTNET}
+      ></sygma-fungible-transfer>`,
+      { parentNode: walletContext }
+    );
+
+    walletContext.dispatchEvent(
+      new WalletUpdateEvent({
+        evmWallet: {
+          address: connectedAddress,
+          providerChainId: 11155111,
+          provider: getMockedEvmWallet().provider
+        }
+      })
+    );
+
+    // Set Source and Destination Networks
+    fungibleTransfer.transferController.onSourceNetworkSelected(cronosNetwork);
+    fungibleTransfer.transferController.onDestinationNetworkSelected(
+      sepoliaNetwork
+    );
+    fungibleTransfer.requestUpdate();
+    await fungibleTransfer.updateComplete;
+
+    const [sygmaSourceNetwork, sygmaDestinationNetwork] =
+      fungibleTransfer.shadowRoot!.querySelectorAll('sygma-network-selector');
+
+    const resourceSelector = fungibleTransfer.shadowRoot!.querySelector(
+      'sygma-resource-amount-selector'
+    ) as ResourceAmountSelector;
+
+    // Wait for sdk init
+    await aTimeout(1000);
```

Comment:
this is not acceptable. Better option would be to mock SDK (init method). and wait for field to update

Reply:
FWIW, I've discussed with @saadjhk and he will open issue to write tests for fungible token transfer with mocks

Reply:
@mpetrunic I added the missing check from your previous comment. Leaving timeout for now

Reply:
Added fix to remove timeouts. Timeouts should never be merged!

Reply:
Added fix to remove timeouts as timeouts should never be merged!

Reply:
@mpetrunic thank you so much for the help with mocking 🙏🏻🙏🏻🙏🏻

Reply:
All good, don't hesitate to ask for help in channel when you encounter stuff like that :)